infraestructure/aws/docs/sns-basic-permissions.md

```diff
@@ -0,0 +1,27 @@
+# SNS Basic Permissions
+
```

Comment:
I would add a subtitle here that clearly explains that this permissions are for sending push notifications. Something like `### Sending push notifications`